docs/blog/posts/voice-service.md

```diff
@@ -0,0 +1,78 @@
+---
+authors:
+  - Mustafa
+category:
+  - API
+  - Components
+date: 2024-09-09
+---
+
+# From Thought to Speech: Introducing AAQ's New Voice Service
+
+We're excited to unveil our new **End-to-End Speech Service** for AAQ! This powerful component brings seamless voice interaction to your Q&A flows. Powered by advanced AI, it integrates Text-to-Speech and Speech-to-Text capabilities, allowing you to easily create dynamic voice experiences through AAQ.
+
+From voice-driven Q&A to automated communication, our service takes voice capabilities to new heights. Get ready to transform how you interact with AAQ!
+
+<!-- more -->
+
+Take a look at it in action integrated with the [Typebot](https://typebot.io/) chatflow manager:
+
+![Voice-workflow](../images/voice-workflow.gif){: .blog-img }
```

Comment:
Can you add a caption underneath this gif saying "A demonstration using STT to ask AAQ --- insert question you asked it here ---". Since there is no sound it's hard to say if it worked well or not/ know what you asked.

Reply:
Hey @MustafaAkolawala - approving but the `---` was just for an example 😅 

I think the sentence should literally read:

A demonstration using the E2E speech pipeline to ask AAQ "What should I eat during pregnancy?"

Reply:
Oh lmao, i will do that quick change then😂